examples/nft/Move.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "nft"
+version = "0.0.1"
+
+[dependencies]
+MoveStdlib = { local = "../../moveos/moveos-stdlib/move-stdlib" }
+MoveosStdlib = { local = "../../moveos/moveos-stdlib/moveos-stdlib" }
+RoochFramework = { local = "../../crates/rooch-framework" }
+
+[addresses]
+creator =  "_"
+std =  "0x1"
+moveos_std =  "0x2"
+rooch_framework =  "0x3"
+
+[dev-addresses]
+creator = "0x42"
```

Comment:
The example project supports two named_address: rooch_examples & $project_dir.
So the creator needs to change to `nft`.

Reply:
#967